com.unity.render-pipelines.high-definition/Runtime/Debug/DebugExposure.shader

```diff
@@ -0,0 +1,555 @@
+Shader "Hidden/HDRP/DebugExposure"
+{
+    HLSLINCLUDE
+
+    #include "Packages/com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/ExposureCommon.hlsl"
+    #include "Packages/com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/HistogramExposureCommon.hlsl"
+    #define DEBUG_DISPLAY
+    #include "Packages/com.unity.render-pipelines.high-definition/Runtime/Debug/DebugDisplay.hlsl"
+    #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/ACES.hlsl"
+    #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Common.hlsl"
+    #include "Packages/com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl"
+
+    #pragma vertex Vert
+    #pragma target 4.5
+    #pragma only_renderers d3d11 playstation xboxone vulkan metal switch
+
+    #define PERCENTILE_AS_BARS 0
+
+    // Contains the scene color post-processed (tonemapped etc.)
+    TEXTURE2D_X(_DebugFullScreenTexture);
+
+    // Tonemap related 
+    TEXTURE3D(_LogLut3D);
+    SAMPLER(sampler_LogLut3D);
+
+    float4 _ExposureDebugParams;
+    float4 _LogLut3D_Params;    // x: 1 / lut_size, y: lut_size - 1, z: contribution, w: unused
+    // Custom tonemapping settings
+    float4 _CustomToneCurve;
+    float4 _ToeSegmentA;
+    float4 _ToeSegmentB;
+    float4 _MidSegmentA;
+    float4 _MidSegmentB;
+    float4 _ShoSegmentA;
+    float4 _ShoSegmentB;
+
+    #define _DrawTonemapCurve   _ExposureDebugParams.x
+    #define _TonemapType        _ExposureDebugParams.y
+
+
+    struct Attributes
+    {
+        uint vertexID : SV_VertexID;
+    };
+
+    struct Varyings
+    {
+        float4 positionCS : SV_POSITION;
+        float2 texcoord : TEXCOORD0;
+    };
+
+    Varyings Vert(Attributes input)
+    {
+        Varyings output;
+        output.positionCS = GetFullScreenTriangleVertexPosition(input.vertexID);
+        output.texcoord = GetNormalizedFullScreenTriangleTexCoord(input.vertexID);
+
+        return output;
+    }
+
+    float3 Tonemap(float3 colorLinear)
+    {
+        if(_TonemapType == 1) // Neutral
```

Comment:
Let's create a true enum (in C# with [HLSL] code genreatin) for the different tonemap type